package.json

```diff
@@ -71,6 +71,7 @@
     "@types/enzyme-adapter-react-16": "^1.0.5",
     "@types/hapi__joi": "^16.0.12",
     "@types/jest": "^25.1.1",
+    "@types/lodash.isequal": "^4.5.5",
```

Comment:
Did @Velenir approve this ;)

Reply:
nope, but it just imports `isEqual` types for the package we already have so I can just import that `fn` and not the entire lib. I think it's up to par with what @Velenir likes to see ;)

Reply:
No, and I'd rather you
```js
import isEqual from 'lodash/isEqual'
```
`lodash.isequal` as well as all other function-specific packages are old

Not to mention I'm not completely sold on using it for deep equality of redux-like state, which should be immutable in the first place

Reply:
fixed